### PR TITLE
fix(navigation): navigation issues fix

### PR DESCRIPTION
**Pull request by 32penkin**
### Please read and mark the following checklist before creating a pull request:

 - [x] I read and followed the [CONTRIBUTING.md](https://github.com/akveo/kittenTricks/blob/master/CONTRIBUTING.md) guide.

 #### Short description of what this resolves:
- Multiple navigation transitions preventing fixed
- Default 'react-navigation' back-button blinking fixed

### CODE REVIEW

File: config/index.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 const path = require('path');
```
Comment (artyorsh) [inline config/index.js]: why is this file indexed?
should not be tracked

please update .gitignore if it is not listed there

Comment (artyorsh) [reply config/index.js]: also it needs to contain production config for users who wants just launch cloned project

File: src/containers/layouts/articles/container/articles.container.tsx [hunk 2]
```diff
@@ -23,7 +23,10 @@ export class ArticlesContainer extends React.Component<NavigationScreenProps, St
   private onItemSelect = (index: number) => {
     const { [index]: selectedItem } = this.data;
 
-    this.props.navigation.dispatch(navigateAction(selectedItem.route));
+    this.props.navigation.navigate({
```
Comment (artyorsh) [inline src/containers/layouts/articles/container/articles.container.tsx:26]: not so bad :)

File: src/core/navigation/routes.tsx [hunk 3]
```diff
@@ -314,6 +319,7 @@ const AppNavigator: NavigationContainer = createStackNavigator({
   ...DarhboardsNavigationMap,
   ...EcommerceNavigationMap,
 }, {
+  headerMode: 'screen',
```
Comment (artyorsh) [inline src/core/navigation/routes.tsx:322]: 👍

**Review by artyorsh — COMMENTED** [review]
Untrack and ignore `./config/index.js`

File: config/index.js (hunk 1 above)
Comment (32penkin) [reply config/index.js]: Done

File: src/containers/layouts/articles/container/articles.container.tsx (hunk 2 above)
Comment (32penkin) [reply src/containers/layouts/articles/container/articles.container.tsx:26]: But not so good.

**Review by artyorsh — APPROVED** [review]

**Pull request merged**